### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**
• Looks better imo, but maybe the logo could be improved
• Removed language sections (such as `scraping/python`) in favour of keeping that info inside `index.html` as there isn't much language-specific to give imo
• Removed `scraping` folder, but I expect that it'll be brought back for challenges

Screenshots:
Screenshot_20220121-191309_Chrome
Screenshot_20220121-191313_Chrome
Screenshot_20220121-191316_Chrome

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,17 +1,240 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <!-- <script src="https://cdn.tailwindcss.com"></script> -->
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Learn Scraping By Doing</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (nate-moo) [inline index.html]: ain't merging bc this

**Review by nate-moo — DISMISSED** [review]
yeet

File: index.html (hunk 1 above)
Comment (AbdullahM0hamed) [reply index.html]: Resolved

**Comment by nate-moo** [comment]
https://user-images.githubusercontent.com/40650681/150594744-c30a0c73-407a-4edd-afb8-d41f2311d98c.mp4

fix before post (AKA turn off theme and dark reader or whatever)

**Comment by AbdullahM0hamed** [comment]
Changed sizing to use percentages (works better across devices)
Made backgrounds white

Mobile:
Screenshot_20220121-205042_Chrome

Desktop mode:
Screenshot_20220121-205053_Chrome

Tested without Chrome's dark mode enabled rip and also on desktop mode

**Pull request merged**